sagemaker-python-sdk/tensorflow_distributed_mnist/tensorflow_local_mode_mnist.ipynb

```diff
@@ -70,7 +70,13 @@
     "from tensorflow.contrib.learn.python.learn.datasets import mnist\n",
     "import tensorflow as tf\n",
     "\n",
-    "data_sets = mnist.read_data_sets('data', dtype=tf.uint8, reshape=False, validation_size=5000)\n",
+    "for attempt in range(5):\n",
+    "    try:\n",
+    "        data_sets = mnist.read_data_sets('data', dtype=tf.uint8, reshape=False, validation_size=5000)\n",
+    "        break\n",
+    "    except IOError as e:\n",
```

Comment:
What IOError here will get? Can you attach stacktrace to this PR?

Reply:
Updated PR with stacktrace